### PR TITLE
/defi page

### PR DESCRIPTION
**Pull request by olaszakos**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: src/pages/defi.tsx [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+      }
+    }
+    window.addEventListener("scroll", onScroll);
+    return () => window.removeEventListener("scroll", onScroll);
+  }, [bgDark, headerHeight]);
+
+  return (
+    <Layout
+      title="DeFi"
+      description={` From fully on-chain order book DEXs, to bridgeless cross-chain
```
Comment (dfx-json) [inline src/pages/defi.tsx:47]: I see an extra whitespace character here

**Review by dfx-json — APPROVED** [review]
LGTM

**Comment by dfx-json** [comment]
Also, nice work @olaszakos. Looks great!

File: src/pages/defi.tsx (hunk 1 above)
Comment (olaszakos) [reply src/pages/defi.tsx]: Good eye, fixed 👍

**Pull request merged**